src/js/components/Search.js

```diff
@@ -211,7 +218,7 @@ export default class Search extends Component {
       if (this.props.onSelect) {
         this.props.onSelect({
           target: this.refs.input || this.refs.control,
-          suggestion: suggestion
+          suggestion: event.target.value
```

Comment:
We do want to pass the suggestion object to onSelect, not the value from the event target.